yaptide/batch/batch_methods.py

```diff
@@ -53,14 +52,16 @@ def submit_job(json_data: dict, cluster: ClusterModel) -> tuple[dict, int]:  # s
 
     submit_script = SUBMIT_SHIELDHIT.format(
         root_dir=job_dir,
-        n_tasks=str(1)
+        n_tasks=str(3),
+        convertmc_version="2.4.0"
```

Comment:
what about:
```
import pymchelper
convertmc_version= pymchelper.__version__
```
?

Reply:
This makes sense, thank you for suggestion